features/exacte_match.feature

```diff
@@ -0,0 +1,171 @@
+  | 2391PH   | 1          | A          |                      | 1672200000009028              |
+  | 2391PH   | 1          | A          |  1                   | 0484200002040487              |
+  | 2391PH   | 1          | A          |  2                   | 0484200002040488              |
+  | 2391PH   | 1          | A          |  3                   | 0484200002040489              |
+  | 2391PH   | 1          | A          |  4                   | 0484200002040490              |
+  | 2391PH   | 1          | A          |  5                   | 0484200002040491              |
+  | 2391PH   | 1          | A          |  6                   | 0484200002040492              |
+  | 2391PH   | 1          | A          |  7                   | 0484200002040493              |
+  | 2391PH   | 1          | A          |  8                   | 0484200002040494              |
+  | 2391PH   | 1          | B          |                      | 1672200000009029              |
+  | 2391PH   | 1          | C          |                      | 1672200000009030              |
+  | 2391PH   | 1          | D          |                      | 1672200000009031              |
+  | 2391PH   | 1          | E          |                      | 1672200000009032              |
+  | 2391PH   | 1          | F          |                      | 1672200000009034              |
+  | 2391PH   | 3          |            |                      | 1672200000005236              |
+  | 2391PH   | 3          | a          |                      | 1672200000005237              |
+  | 2391PH   | 5          |            |                      | 1672200000005239              |
+  | 2391PH   | 7          |            |                      | 1672200000009548              |
+  | 2391PH   | 9          |            |                      | 1672200000008470              |
+  | 2391PH   | 11         |            |                      | 1672200000005233              |
+  | 2594AN   | 265        |            |                      | 0518200000390367              |
+  | 2594AN   | 265        | A          |                      | 0518200000452568              |
+  | 2594AN   | 265        | A          | 1                    | 0518200000452570              |
+  | 2594AN   | 265        | A          | 2                    | 0518200000620759              |
+  | 2594AN   | 265        | A          | 3                    | 0518200000625989              |
+  | 2594AN   | 265        | B          |                      | 0518200000454095              |
+  | 2594AN   | 265        | B          | 1                    | 0518200001647713              |
+  | 2594AN   | 265        | B          | 2                    | 0518200001647715              |
+  | 2594AN   | 265        | B          | 3                    | 0518200001647717              |
+  | 2594AN   | 265        | B          | 3                    | 0518200001775151              |
```

Comment:
Zijn in regel 55 bewust dezelfde zoekparameters gedefinieerd als in regel 54?
Deze records worden in de scenario's niet gebruikt dus ik vermoed van niet.

Reply:
Tabel opgeschoond, maar niet alle ongebruikte voorbeelden verwijderd.